pkgs/tools/wayland/sov/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, meson
+, ninja
+, pkg-config
+, wayland
+, wayland-protocols
+, wayland-scanner
+, freetype
+, linuxHeaders
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sov";
+  version = "unstable-2022-04-26";
+
+  src = fetchFromGitHub {
+    owner = "milgra";
+    repo = "sov";
+    rev = "5bed5299396eb5bb334fda83b8b84301ffa8ae51";
```

Comment:
Any specific reason to not use [v0.63](https://github.com/milgra/sov/releases/tag/0.63)?

Reply:
Nope, that was released after I made this PR. Updated!